.github/workflows/bench.yml

```
@@ -0,0 +1,60 @@
+name: Benchmarks
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  bench:
+    name: regression
+    permissions:
+      checks: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: bencherdev/bencher@main
+      - name: Use Rust Stable
+        uses: dtolnay/rust-toolchain@stable
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20.x
+          cache: npm
+      - name: npm install
+        run: npm ci --prefer-offline --no-audit --no-fund
+      - name: Install Benchmark Dependencies
+        working-directory: ./bench
+        run: npm install
+      - name: Build Benchmark
+        working-directory: ./bench
+        run: npm run build
+      - name: Track base branch benchmarks with Bencher
+        if: github.ref_name == 'main'
+        run: bencher run
+          --project neon
+          --token '${{ secrets.BENCHER_API_TOKEN }}'
+          --branch main
+          --testbed ubuntu-latest
+          --adapter json
+          --github-actions '${{ secrets.GITHUB_TOKEN }}'
+          npm run benchmark
```

We need `--silent` on these commands. Otherwise, `npm` will write the command being executed to `stdout` and break JSON parsing.

```suggestion
          npm run --silent benchmark
```